it/test/itbase/ItSpecBase.scala

```diff
@@ -41,19 +40,9 @@ trait ItSpecBase extends AnyFreeSpec with Matchers with ScalaFutures with Option
   def guiceApplicationBuilder(): GuiceApplicationBuilder =
     new GuiceApplicationBuilder()
       .configure("metrics.enabled" -> false)
+      .configure("play.http.router" -> "testOnlyDoNotUseInAppConf.Routes")
```

Comment:
Do the tests fail without this, out of interest? Wouldn't have expected them to be necessary in the integration tests.